main.go

```diff
@@ -23,15 +24,16 @@ import (
 var (
 	success = color.New(color.FgGreen)
 	fail    = color.New(color.FgHiRed)
-	ignore = true
+	ignore = flag.Bool("ignore",false,"ignore packages with no test files")
 )
 
 const paletteEnv = "GOTEST_PALETTE"
 
 func main() {
+	flag.Parse()
 	setPalette()
 	enableOnCI()
-	os.Exit(gotest(os.Args[1:]))
+	os.Exit(gotest(os.Args[2:]))
```

Comment:
```suggestion
	os.Exit(gotest(os.Args[1:]))
```

We don't know that the flag is present or that it's one of the first two arguments. Additionally, `go test` deals with unknown flags gracefully, so there's no reason to avoid forwarding it.